xrpl/asyncio/transaction/reliable_submission.py

```diff
@@ -113,3 +118,110 @@ async def send_reliable_submission(
     return await _wait_for_final_transaction_outcome(
         transaction_hash, client, prelim_result, 0
     )
+
+
+def _is_signed(transaction: Transaction) -> bool:
```

Comment:
Does the `Transaction` object not have some sort of `is_signed` member function? I thought it did...

Reply:
IMO this would be better suited as an instance method on the `Transaction` class.